source/Calamari.Common/Features/Scripting/WindowsPowerShell/Bootstrap.ps1

```diff
@@ -292,6 +292,39 @@ function Write-Verbose
 	}
 }
 
+<#
+.SYNOPSIS
+Writes text to stderr when running in a regular console window,
+to the host''s error stream otherwise.
+
+.DESCRIPTION
+Writing to true stderr allows you to write a well-behaved CLI
+as a PS script that can be invoked from a batch file, for instance.
+
+Note that PS by default sends ALL its streams to *stdout* when invoked from
+cmd.exe.
+
+This function acts similarly to Write-Host in that it simply calls
+.ToString() on its input; to get the default output format, invoke
+it via a pipeline and precede with Out-String.
+
+#>
+function Write-StdErr {
+	param ([PSObject] $InputObject)
+	$outFunc = if ($Host.Name -eq 'ConsoleHost') {
+		[Console]::Error.WriteLine
+	} else {
+		$host.ui.WriteErrorLine
+	}
+	if ($InputObject) {
+		[void] $outFunc.Invoke($InputObject.ToString())
+	} else {
+		[string[]] $lines = @()
+		$Input | % { $lines += $_.ToString() }
+		[void] $outFunc.Invoke($lines -join "`r`n")
+	}
+}
+
```

Comment:
I think that's basically saying if you run `Write-StdErr` with no arg, it'll print a new line. My main concern with removing it would be that it might not meet other devs' expectations if nothing happens when it's used.

Also technically this function would now be available to customers to use in their own scripts - but it's not advertised anywhere so the only way they would know is if they inspected this code.

Reply:
I've removed it - just used `Write-Warning` instead.